lib/signaling/v2/iceconnectionmonitor.js

```diff
@@ -43,8 +43,10 @@ class IceConnectionMonitor {
     });
   }
 
-  _getActivePairStat(stats) {
-    const statsArray = Array.from(stats.values());
```

Comment:
Will this cause an issue in a non-Citrix environment?

Reply:
No, it will not. I reverted this change, too.